packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/vsphere-users/add/vsphere-user-add.html

```diff
@@ -1,6 +1,6 @@
 <div
     data-wizard
-    data-wizard-on-cancel="$ctrl.goBack.bind($ctrl)"
+    data-wizard-on-cancel="$ctrl.onCancel.bind($ctrl)"
     data-wizard-on-finish="$ctrl.addUser.bind($ctrl)"
```

Comment:
same as https://github.com/ovh/manager/pull/8158/files#r993037035
```suggestion
    data-wizard-on-cancel="$ctrl.boundOnCancel"
    data-wizard-on-finish="$ctrl.boundAddUser"
```

Reply:
It's applied.